bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Verify/VerifyElection.cs

```diff
@@ -0,0 +1,1087 @@
+        using var ac0 = BigMath.PowModP(selection.Ciphertext.Pad, selection.Proof.ZeroChallenge);
+        using var gv0ac0 = BigMath.MultModP(gv0, ac0);
+        var consistent_gv0 = selection.Proof.ZeroPad.Equals(gv0ac0);
+        results.Add(new VerificationResult(consistent_gv0, $"      - Verification 4.1: 𝑎0 = 𝑔^𝑣0 mod 𝑝 ⋅ 𝛼^𝑐0 mod 𝑝"));
+
+        // Verification 4.2 - 𝑎1 = 𝑔^𝑣1 mod 𝑝 ⋅ 𝛼^𝑐1 mod 𝑝
+        using var gv1 = BigMath.PowModP(constants.G, selection.Proof.OneResponse);
+        using var ac1 = BigMath.PowModP(selection.Ciphertext.Pad, selection.Proof.OneChallenge);
+        using var gv1ac1 = BigMath.MultModP(gv1, ac1);
+        var consistent_gv1 = selection.Proof.OnePad.Equals(gv1ac1);
+        results.Add(new VerificationResult(consistent_gv1, $"      - Verification 4.2: 𝑎1 = 𝑔^𝑣1 mod 𝑝 ⋅ 𝛼^𝑐1 mod 𝑝"));
+
+        // Verification 4.3 - 𝑏0 = 𝐾^𝑣0 mod 𝑝 ⋅ 𝛽^𝑐0 mod 𝑝
+        using var kv0 = BigMath.PowModP(context.ElGamalPublicKey, selection.Proof.ZeroResponse);
+        using var bc0 = BigMath.PowModP(selection.Ciphertext.Data, selection.Proof.ZeroChallenge);
+        using var kv0bc0 = BigMath.MultModP(kv0, bc0);
+        var consistent_kv0 = selection.Proof.ZeroData.Equals(kv0bc0);
+        results.Add(new VerificationResult(consistent_kv0, $"      - Verification 4.3: 𝑏0 = 𝐾^𝑣0 mod 𝑝 ⋅ 𝛽^𝑐0 mod 𝑝"));
+
+        // Verification 4.4 - 𝑏1 = 𝐾^w1 mod 𝑝 ⋅ 𝛽^𝑐1 mod 𝑝
+        using var w1 = BigMath.SubModQ(selection.Proof.OneResponse, selection.Proof.OneChallenge);
+        using var kw1 = BigMath.PowModP(context.ElGamalPublicKey, w1);
+        using var bc1 = BigMath.PowModP(selection.Ciphertext.Data, selection.Proof.OneChallenge);
+        using var kw1bc1 = BigMath.MultModP(kw1, bc1);
+        var consistent_kw1 = selection.Proof.OneData.Equals(kw1bc1);
+        results.Add(new VerificationResult(consistent_kw1, $"      - Verification 4.4: 𝑏1 = 𝐾^w1 mod 𝑝 ⋅ 𝛽^𝑐1 mod 𝑝"));
+
+        // Verification 4.5 - c = H(04,Q,K,α,β,a0,b0,a1,b1)
+        // using var recomputedHash = Hash.HashElems(
+        //     Hash.Prefix_SelectionProof,
+        //     context.CryptoExtendedBaseHash,
+        //     context.ElGamalPublicKey,
+        //     selection.Ciphertext.Pad,
+        //     selection.Ciphertext.Data,
+        //     selection.Proof.ZeroPad,
+        //     selection.Proof.ZeroData,
+        //     selection.Proof.OnePad,
+        //     selection.Proof.OneData);
+        // var consistent_hash = selection.Proof.Challenge.Equals(recomputedHash);
+        // results.Add(new VerificationResult(consistent_hash, $"      - Verification 4.5: c = H(04,Q,K,α,β,a0,b0,a1,b1)"));
+
+        // TODO: Issue #433 - E.G. 2.0 - implement
+        // https://github.com/microsoft/electionguard-core2/issues/433
+        results.Add(new VerificationResult(IsValidwithKnownSpecDeviations, "- Verification 4.5: TODO: E.G. 2.0 - implement"));
+
+
+        // Verification 4.A
+        var inBoundsAlpha = selection.Ciphertext.Pad.IsInBounds();
```

Comment:
where the verification numbering is off between 1.53 and 2.0.0 i kept the 2.0.0 headings but replaced the body statements with the 1.53 numbering.